website/docs/docs/cloud/manage-access/enterprise-permissions.md

```diff
@@ -4,12 +4,7 @@ id: "enterprise-permissions"
 description: "Permission sets for Enterprise plans."
 ---
 
-:::info Enterprise Feature
-
-This guide describes a feature of the dbt Cloud Enterprise plan.
-If you're interested in learning more about an Enterprise plan, contact us at sales@getdbt.com.
-
-:::
+<Snippet src="available-enterprise-only" />
```

Comment:
Wonder if we should switch to partials:
`import SetUpPages from '/snippets/_available-enterprise-only.md';`

Reply:
addressed in https://github.com/dbt-labs/docs.getdbt.com/pull/3616